Reconstruction/CRDEcalRec/src/Algorithm/TrackExtrapolatingAlg.cpp

```diff
@@ -16,9 +16,9 @@ StatusCode TrackExtrapolatingAlg::ReadSettings(Settings& m_settings){
   //Initialize parameters
   // if(settings.map_floatPars.find("Par1")==settings.map_floatPars.end()) settings.map_floatPars["Par1"] = 0;
   if(settings.map_floatPars.find("innermost_distance")==settings.map_floatPars.end()) 
-    settings.map_floatPars["innermost_distance"] = 1865;
+    settings.map_floatPars["innermost_distance"] = 1860;
   if(settings.map_floatPars.find("outermost_distance")==settings.map_floatPars.end()) 
-    settings.map_floatPars["outermost_distance"] = 2400;
+    settings.map_floatPars["outermost_distance"] = 2140;
   if(settings.map_intPars.find("Nlayers")==settings.map_intPars.end()) 
     settings.map_intPars["Nlayers"] = 28;
   if(settings.map_floatPars.find("layer_width")==settings.map_floatPars.end())

```

Comment:
For the following "normal_vec" value I missed last time: if they are necessary to be added in settings? (i.e. will you change them in any condition?)
If they are just normal initial value, you can write them in code by hardcoding.

Reply:
These "normal_vec" are vectors that are perpendicular to each module.
They will be changed if the number of module changes or the whole ECAL rotate an angle.
If the ECAL with 8 modules do not change, "normal_vec" can be written as hardcode

Reply:
Then you can use the module number as an option, and hardcode these values in the algorithm. 
```
if(settings.map_floatPar["Nmodule"]==8){
  xxxx;
}
```